src/app/interaction-model/Model.h

```diff
@@ -43,17 +43,17 @@ class Model : public AttributeTreeIterator
 public:
     virtual ~Model() = default;
```

Comment:
Again: why is this called InteractionModel::Model when it's a "data model"?

Reply:
I would like to do renames as part of a separate PR that just renames. It seems to need file and directory shuffling and combining that with documentation changes is rough.